### PR TITLE
[debugger] Pass retry timeout to runtime

### PR DESCRIPTION
**Pull request by thaystg**

### CODE REVIEW

**Comment by jonathanpeppers** [comment]
monodroid side is: https://github.com/xamarin/monodroid/pull/1293

**Review by grendello — APPROVED** [review]

**Comment by thaystg** [comment]
These failures are related?

File: .external [hunk 1]
```diff
@@ -1,2 +1,2 @@
-xamarin/monodroid:main@602aca98245744882a129206b79b5a5e093dae44
+xamarin/monodroid:thays_support_debug_server_n@4bc0f7acf04668e1fa1a6fca63ea68ca9b2d40ec
```
Comment (jonathanpeppers) [inline .external:1]: We should merge the xamarin/monodroid side first, then update this line to monodroid/main.

**Review by jonathanpeppers — CHANGES_REQUESTED** [review]
I reran some of the failing tests, they do not look related to me.

File: .external (hunk 1 above)
Comment (thaystg) [reply .external]: done.

**Review by jonathanpeppers — APPROVED** [review]

**Comment by thaystg** [comment]
@jonathanpeppers can we merge this one?

**Comment by jonathanpeppers** [comment]
I think the build is too old to see the test failures. I will start it again.

**Pull request merged**